redis/src/main/scala/zio/redis/Output.scala

```diff
@@ -813,4 +813,35 @@ object Output {
         case other                          => throw ProtocolError(s"$other isn't an integer >= -1")
       }
   }
+
+  case object ClientInfoOutput extends Output[ClientInfo] {
+    protected def tryDecode(respValue: RespValue)(implicit _codec: Codec): ClientInfo = {
```

Comment:
```suggestion
    protected def tryDecode(respValue: RespValue)(implicit codec: Codec): ClientInfo = {
```

Reply:
I have updated this in the last commit (merged with master)